app/models/host/managed_extensions.rb

```diff
@@ -32,7 +32,8 @@ def setReboot
     # continue. If flash is implemented for new hosts (http://projects.theforeman.org/issues/10559)
     # we can report the error to the user perhaps.
     true
-  rescue ::Foreman::Exception
+  rescue => e
```

Comment:
I just wanted to catch them all, I had nothing in particular in mind. Shall I revert it?